style.css

```diff
@@ -182,4 +182,84 @@ h4 {
     border: none;
     background-color: #FFA62B;
     transition: ease-in 0.3s;
+}
+
+@media screen and (max-width: 640px) {
+
```

Comment:
Inside media query you should write only the styles that are specific for current screen size. Don't repeat styles that are already set up before